playbooks/storage.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: localhost
+  vars:
+    storage_role: storage-demo
+  connection: local
+  gather_facts: False
+  # unset http_proxy. required for running in the CI
+  environment:
+    http_proxy: ""
+  roles:
+    - role: "{{ storage_role }}"
```

Comment:
Shouldn't we first call `storage-demo-nodeconfig`?

Reply:
The order isn't important.  Both roles need to be applied before the storage can actually be used.